library/general/src/modules/Arch.rb

```diff
@@ -516,6 +516,23 @@ def is_wsl
       SCR.Read(path(".target.string"), "/proc/sys/kernel/osrelease").to_s.downcase.include?("-microsoft")
     end
 
+    # Check whether managing another target system. The target system location
+    # is then specified in the Installation.destdir value.
+    #
+    # @note Technically the management can run in a chroot or in a Linux
+    #   container (Docker, Podman,...), or using a Live medium... It does not
+    #   check how it is implemented.
+    #
+    #   The main goal is to distinguish between running in a container and managing
+    #   the container itself or running in a container and managing the host
+    #   system which runs the container.
+    #
+    # @return [Boolean] true if managing another system in chroot/container
+    def chroot_management
+      target_dir = ENV["YAST_TARGET_DIR"] || ""
+      File.directory?(target_dir)
```

Comment:
question is how it differs from from common installation where it also from certain point you manage chrooted env.
What about re-using already existing pieces and 
1. during initialize if YAST_TARGET_DIR is set then switch SCR to that chroot
2. for detection use generic `scr_chrooted?` https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/wfm.rb#L119

so this way it should work same as installation?